src/libraries/System.Private.CoreLib/src/ILLink/ILLink.LinkAttributes.Shared.xml

```diff
@@ -219,6 +219,9 @@
     <type fullname="System.Runtime.Versioning.TargetPlatformAttribute">
       <attribute internal="RemoveAttributeInstances" />
     </type>
+    <type fullname="System.Runtime.Versioning.TargetFrameworkAttribute">
```

Comment:
This is used here:

https://github.com/dotnet/runtime/blob/12a8819eee9865eb38bca6c05fdece1053102854/src/libraries/System.Private.CoreLib/src/System/AppContext.cs#L30

Reply:
Right but that's on the entry assembly which is never trimmed and these settings are inside AggressiveAttributeTrimming section. Although I don't have strong preference on this one

Reply:
> the entry assembly which is never trimmed

Hopefully won't be true in the future. The current "by default we don't touch user code" is really a hacky workaround for the lack of ability to correctly trim certain patterns (serializers...).

Reply:
ok, removed the type